components/support/SupportLayerItem.vue

```diff
@@ -99,43 +99,45 @@ export default {
         wmsBaseUrl() {
             let wmsUrl = ''
             if (this.layer.layer_type === 'wms') {
-                if (this.layer.filters) {
-                    wmsUrl =
-                        this.layer.wms.geoserver.wms_url +
-                        '&env=percentage:' +
-                        this.layer.opacity / 100
-
-                    const filters = this.layer.filters
-                    const keys = Object.keys(filters)
-
-                    if (keys.length) {
-                        wmsUrl += '&CQL_FILTER='
-
-                        keys.forEach(element => {
-                            let el = ''
-                            if (Array.isArray(filters[element])){
-                                el = JSON.stringify(filters[element]).slice(1, -1)
-                            } else {
-                                el = filters[element]
-                            }
-                            wmsUrl += element + ' IN (' + el + ') AND '  
-                        });
-                        wmsUrl = wmsUrl.slice(0, -4)
+                const filters = this.layer.filters
+                
+                if (filters.co_cr || filters.co_funai) {
+                    let [aliasCoordenacao, aliasTi] = this.layer.layer_filters // Destructuring filter alias
+
+                    wmsUrl = `${
+                        this.layer.wms.geoserver.wms_url
+                    }&env=percentage:${this.layer.opacity / 100}`
+
+                    if (filters.co_cr.length && filters.co_funai.length) {
+                        let result = filters.co_cr.join(',')
+                        let result2 = filters.co_funai.join(',')
+
+                        wmsUrl += `&CQL_FILTER=${aliasCoordenacao.filter_alias} IN (${result}) AND ${aliasTi.filter_alias} IN (${result2})`
+                        
+                        this.$nextTick(() => {
+                            this.$refs.wmsLayer.mapObject.setUrl(wmsUrl)
+                        })
+                        return wmsUrl
                     }
-                } else {
-                    wmsUrl =
-                        this.layer.wms.geoserver.wms_url +
-                        '&env=percentage:' +
-                        this.layer.opacity / 100
+                    if (filters.co_cr.length) {
+                        let list_coord = filters.co_cr.join(',')
+                        wmsUrl += `&CQL_FILTER=${aliasCoordenacao.filter_alias} IN (${list_coord})`
+                    }
+                    if (filters.co_funai.length) {
```

Comment:
missing start_date and end_date filters.

Reply:
Resolved